server/parsec/config.py

```diff
@@ -115,6 +115,9 @@ class BackendConfig:
     forward_proto_enforce_https: tuple[str, str] | None
     server_addr: ParsecAddr | None
 
+    # If log level is DEBUG, we sometime want to replace some INFO logs by
+    # equivalent ones with more details.
+    log_level_is_debug: bool
```

Comment:
Why not simply `log_level`?

This would be more flexible than a single boolean.

Reply:
YAGNI kid, you should focus more on KISS :trollface: